lib/querystring.js

```diff
@@ -186,12 +208,23 @@ function encodeStringified(v, encode) {
   return '';
 }
 
-
+/**
+ * @param {string | number | boolean | null} v 
+ * @param {(v: string) => string} encode 
+ * @returns {string}
+ */
 function encodeStringifiedCustom(v, encode) {
   return encode(stringifyPrimitive(v));
 }
 
-
+/**
+ * 
+ * @param {Record<string, string | number | boolean | ReadonlyArray<string> | ReadonlyArray<number> | ReadonlyArray<boolean> | null>} obj 
```

Comment:
```suggestion
 * @param {Record<string, string | number | boolean
 * | ReadonlyArray<string | number | boolean> | null>} obj
```
Max line length -> 80 + Lint.